packages/gcloud-logger/src/Logger.ts

```diff
@@ -129,9 +131,12 @@ export class Logger {
 
   private stringify(message: any) {
     const excludeFn = (key: string, value: string) => {
-      if (this.excludeKeys.indexOf(key) !== -1) {
+      if (this.excludeKeys.includes(key)) {
```

Comment:
```suggestion
      if (this.excludeKeys.includes(key)) {
```
I believe `indexOf` is faster

Reply:
@adam187 benchmark tells include is faster

https://www.measurethat.net/Benchmarks/ShowResult/119361

Reply:
I think it depends on the data type of array elements, btw it's no a big deal